src/openseadragon.js

```diff
@@ -2113,11 +2129,21 @@ function OpenSeadragon( options ){
             try {
                 request.open( "GET", url, true );
 
+                if (responseType) {
+                    request.responseType = responseType;
+                }
+
+                if (headers) {
+                    Object.keys(headers).forEach(function (headerName) {
```

Comment:
We avoid using forEach for compatibility with IE8. This code will probably never be called with IE8 but better safe than sorry. We could implement a `$.forEach(array, callback)` method in openseadragon.js.